kyo-playwright/shared/src/main/scala/kyo/Image.scala

```diff
@@ -0,0 +1,130 @@
+package kyo
+
+import java.util.Base64
+
+/** Represents an image with utility methods for manipulation and display.
+  *
+  * This class provides functionality for:
+  *   - Writing images to files in binary or base64 format
+  *   - Converting between binary and base64 representations
+  *   - Rendering images directly to compatible terminal emulators
+  *
+  * Images can be created from binary data or base64 strings using the companion object's factory methods.
+  */
+case class Image(data: Array[Byte]):
+
+    /** Writes the image to a file in binary format.
+      *
+      * @param path
+      *   The file path as a string
+      * @return
+      *   Unit wrapped in IO effect
+      */
+    def writeFileBinary(path: String)(using Frame): Unit < IO =
+        writeFileBinary(Path(path))
+
+    /** Writes the image to a file in binary format.
+      *
+      * @param path
+      *   The file path as a Path object
+      * @return
+      *   Unit wrapped in IO effect
+      */
+    def writeFileBinary(path: Path)(using Frame): Unit < IO =
+        path.writeBytes(binary.unsafeArray)
+
+    /** Writes the image to a file in base64 format.
+      *
+      * @param path
+      *   The file path as a string
+      * @return
+      *   Unit wrapped in IO effect
+      */
+    def writeFileBase64(path: String)(using Frame): Unit < IO =
+        writeFileBase64(Path(path))
+
+    /** Writes the image to a file in base64 format.
+      *
+      * @param path
+      *   The file path as a Path object
+      * @return
+      *   Unit wrapped in IO effect
+      */
+    def writeFileBase64(path: Path)(using Frame): Unit < IO =
+        path.write(base64)
+
+    /** Converts the image data to an immutable array of bytes.
+      *
+      * @return
+      *   The image data as an IArray[Byte]
+      */
+    def binary: IArray[Byte] = IArray(data*)
```

Comment:
```suggestion
    def binary: IArray[Byte] = IArray.unsafeFromArray(data)
```
https://www.scala-lang.org/api/current/scala/IArray$.html#unsafeFromArray-957